javagen/src/test/java/com/azure/autorest/mapper/ClientMethodMapperAccessor.java

```diff
@@ -0,0 +1,16 @@
+package com.azure.autorest.mapper;
+
+import com.azure.autorest.extension.base.model.codemodel.Operation;
+import com.azure.autorest.model.clientmodel.IType;
+
+/**
+ * @author xiaofeicao
+ * @createdAt 2021-12-29 15:14
+ */
+public class ClientMethodMapperAccessor extends ClientMethodMapper{
```

Comment:
Why do we need to extend the `ClientMethodMapper` to have a static method in it?

Reply:
It's due to test case invocation here https://github.com/XiaofeiCao/autorest.java/blob/2136e157a43171f7375db7e83ec3aff96a90324d/javagen/src/test/java/com/azure/autorest/JavagenUnitTests.java#L78
Original method is `protected` so I subclassed to create a `public` method which invokes the original method, to allow for test case invocation.